client/landing/stepper/declarative-flow/internals/types.ts

```diff
@@ -6,6 +6,9 @@ import React from 'react';
 export type NavigationControls = {
 	/**
 	 * Call this function if you want to go to the previous step.
+	 *
+	 * Please don't change the type of this function to add parameters. Passing data should strictly happen through the `submit` function.
+	 * If you disagree with this, please ping Vertex Team.
```

Comment:
If you have a link to discussion that's typically more long-lasting than a team name mention, which come and go over the long years :-)